cloudferry/lib/utils/mysql_connector.py

```diff
@@ -36,6 +41,29 @@ def get_db_host(cloud_config):
     return db_host
 
 
+def dump_db(cloud, database=ALL_DATABASES):
+    cmd = "mysqldump {database} --user={user}"
```

Comment:
This command produces tons of unnecessary logs, please consider calling `mysqldump` remotely